package.json

```diff
@@ -106,16 +104,8 @@
     "react-virtualized": "^9.22.3"
   },
   "dependencies": {
-    "lodash.debounce": "4.0.8",
-    "regenerator-runtime": "^0.13.9"
+    "lodash.debounce": "4.0.8"
   },
-  "browserslist": [
```

Comment:
why does this need to be removed? we were already excluding IE 11 and earlier

Reply:
This is only used by the preset-env